sources/tests/Makefile

```diff
@@ -22,3 +23,8 @@ e2e-tests-queue-consumers:
 	@echo "+\n++ Make: Starting containers for Queue Consumers E2E tests...\n+"
 	@docker-compose run --name queue-consumers-e2e-$(BUILD_REF) --rm queue-consumers npm run test:e2e:queue-consumers
 	@docker-compose down --rmi all -v
+
+e2e-tests-workflow:
+	@echo "+\n++ Make: Starting containers for camunda workflow E2E tests...ON TESTS \n+"
```

Comment:
Why "ON TESTS" was appended to the message?

Reply:
That came from copy paste using the code of e2e-tests-workers. 

Btw e2e-tests-api and e2e-tests-workers also have it. Should we remove there also?